### PR TITLE
update etcd unified version to 3.1.10

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:
update etcd unitied version to `3.1.10`
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```
bump etcd version to 3.1.10
```

### CODE REVIEW

**Comment by mikedanese** [comment]
@kubernetes/sig-api-machinery-pr-reviews

File: hack/make-rules/test-cmd-util.sh [hunk 1]
```diff
@@ -2539,15 +2539,15 @@ run_rc_tests() {
   # Pre-condition: 2 replicas
   kube::test::get_object_assert 'rc frontend' "{{$rc_replicas_field}}" '2'
   # Command
-  ! kubectl scale --current-replicas=3 --replicas=2 replicationcontrollers frontend "${kube_flags[@]}"
+  kubectl scale --current-replicas=3 --replicas=2 replicationcontrollers frontend "${kube_flags[@]}"
```
Comment (liggitt) [inline hack/make-rules/test-cmd-util.sh]: this looks unrelated... why is it in this PR?

File: hack/make-rules/test-cmd-util.sh [hunk 2]
```diff
@@ -2539,15 +2539,15 @@ run_rc_tests() {
...
   # Post-condition: nothing changed
   kube::test::get_object_assert 'rc frontend' "{{$rc_replicas_field}}" '2'
 
   ### Scale replication controller frontend with replicas only
   # Pre-condition: 2 replicas
   kube::test::get_object_assert 'rc frontend' "{{$rc_replicas_field}}" '2'
   # Command
-  kubectl scale  --replicas=3 replicationcontrollers frontend "${kube_flags[@]}"
+  kubectl scale --current-replicas=2  --replicas=3 replicationcontrollers frontend "${kube_flags[@]}"
```
Comment (liggitt) [inline hack/make-rules/test-cmd-util.sh]: same here

**Comment by zouyee** [comment]
gentle ping @liggitt

**Comment by liggitt** [comment]
Making the versions consistent looks good to me. Does the etcd cleanup image actually exist with that version?

Needs an approved under the cluster dir

**Comment by zouyee** [comment]
@roberthbailey  PTAL

**Comment by zouyee** [comment]
https://github.com/kubernetes/kubernetes/issues/58578

**Comment by liggitt** [comment]
should be picked to 1.9, right? and then bumped to 3.2.x along with the server changes in https://github.com/kubernetes/kubernetes/pull/58645, right?

**Comment by tallclair** [comment]
@ixdy - This image needs a push. Who usually handles this?

**Comment by ixdy** [comment]
there isn't a particular usual person for addon images, though I often end up doing it.

that said, I'll maybe nominate @jpbetz here, since this is related to etcd.

**Comment by jpbetz** [comment]
@ixdy Sounds good. I can own this.

gcr.io/google-containers/etcd-empty-dir-cleanup:3.1.10.0 pushed.
